### PR TITLE
Add SQS blogpost

### PR DESCRIPTION
**Pull request by t4lz**

### CODE REVIEW

File: content/en/blog/sqs-splitting/index.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+lastmod: 2024-10-01T06:00:00+00:00
+draft: false
+weight: 50
+contributors: ["Tal Zwick"]
+---
+
+One of the reasons [mirrord for Teams](https://mirrord.dev/docs/overview/teams/) is a gamechanger for many 
+organizations is that it makes it possible for a whole team to work with one shared cloud environment. They don’t 
+need to run the whole environment for each developer, they don’t need to have their entire architecture running on 
+their laptops, and they do not need to redeploy to Kubernetes to test new code or debug it. Developers just press
```
Comment (gememma) [inline content/en/blog/sqs-splitting/index.md:23]:
```suggestion
their laptops, and they don't need to redeploy to Kubernetes to test new code or debug it. Developers just press 
```

**Review by gememma — COMMENTED** [review]
image
This is very picky but the arrow connecting to the upper left box here doesn't connect at the vertical middle of the box

image
I don't think the laptop icons here are very legible (do they need to be there?) and the filter icons are too close in colour to the box outline - they need to be visually separate. I personally don't think the font is readable at this size either but I understand that it needs to be consistent

**Comment by t4lz** [comment]
Changed the diagrams @gememma

**Review by gememma — APPROVED** [review]
Very nice 🏅

**Pull request merged**